jmslib/src/main/java/com/redhat/mqe/lib/JmsMessageFormatter.java

```diff
@@ -148,6 +148,21 @@ public void addFormatJMS20(Message msg, Map<String, Object> result) throws JMSEx
         return result;
     }
 
+    /**
+     * Print message in interoperable way for comparing with other clients but without body.
+     * This is suitable for large messages
+     */
+    public Map<String, Object> formatMessageAsLargeInterop(Message msg, boolean hashContent) throws JMSException {
+        Map<String, Object> result = formatMessage(msg, hashContent);
+        addFormatInterop(msg, result);
+        result.put("id", removeIDprefix((String) result.get("id")));
+        result.put("user-id", removeIDprefix((String) result.get("user-id")));
+        result.put("correlation-id", removeIDprefix((String) result.get("correlation-id")));
+        result.put("group-sequence", changeMinusOneToZero((long) result.get("group-sequence")));
+        result.put("content", "<omitted due to large message size>");
```

Comment:
@jdanekrh if you have some better idea of this. 
Maybe some hash of content might be better?